integrations/activemq/documentation.yaml

```diff
@@ -0,0 +1,127 @@
+      lowercaseOutputName: true
+      lowercaseOutputLabelNames: true
+      blacklistObjectNames:
+        - "org.apache.activemq:clientId=*,*"
+      whitelistObjectNames:
+        - "org.apache.activemq:destinationType=Queue,*"
+        - "org.apache.activemq:destinationType=Topic,*"
+        - "org.apache.activemq:type=Broker,brokerName=*"
+        - "org.apache.activemq:type=Topic,brokerName=*"
+
+      rules:
+      - pattern: org.apache.activemq<type=Broker, brokerName=(\S*), destinationType=Queue, destinationName=(\S*)><>(\w+)
+        name: activemq_queue_$3
+        attrNameSnakeCase: true
+        labels:
+          destination: $2
+
+      - pattern: org.apache.activemq<type=Broker, brokerName=(\S*), destinationType=Topic, destinationName=(\S*)><>(\w+)
+        name: activemq_topic_$3
+        attrNameSnakeCase: true
+        labels:
+          destination: $2
+
+      - pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>CurrentConnectionsCount
+        name: activemq_connections
+        type: GAUGE
+
+      - pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>Total(.*)Count
+        name: activemq_$2_total
+        type: COUNTER
+
+      - pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>(.*)PercentUsage
+        name: activemq_$2_usage_ratio
+        type: GAUGE
+        valueFactor: 0.01
+  ---
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: activemq
+  spec:
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: activemq
+    template:
+      metadata:
+        labels:
+  +       app.kubernetes.io/name: activemq
+      spec:
+        containers:
+        - name: activemq
+          image: rmohr/activemq:5.15.9-alpine
+  +     ports:
+  +       - containerPort: 1099
+  +         name: jmx
+  +     env:
+  +       - name: ACTIVEMQ_JMX
+  +         value: "1099"
+  +       - name: ACTIVEMQ_OPTS
+  +         value: "-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=1099 -Dcom.sun.management.jmxremote.rmi.port=1099 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
+  +     - name: exporter
+  +       image: bitnami/jmx-exporter:0.17.0
+  +       command:
+  +         - java
+  +         - -jar
+  +         - jmx_prometheus_httpserver.jar
+  +       args:
+  +         - "9000" 
+  +         - config.yaml
+  +       ports:
+  +       - containerPort: 9000
+  +         name: prometheus
+  +       volumeMounts:
+  +       - mountPath: /opt/bitnami/jmx-exporter/config.yaml
+  +         subPath: config.yaml
+  +         name: activemq-exporter
+  +     volumes:
+  +     - name: activemq-exporter
+  +       configMap:
+  +         name: activemq-exporter
+  +         items:
+  +         - key: config.yaml
+  +           path: config.yaml
+additional_install_info: |
+  These instructions assume you already have a working {{app_name_short}}
+  installation and want to modify it to include an exporter.
```

Comment:
Given that the template will already say "To install {{exporter_literal}} as a sidecar to {{app_name_short}}, modify your {{app_name_short}} configuration as shown in the following
example:"

Maybe this is redundant. 

Let's make the standard that we add a clarifying paragraph like this only when (as a bonus, as with the Redis case) followed immediately by additional detail for the other use case ("To set up {{app_name_short}} and {{exporter_name}} together from scratch, use this Helm chart...").